lib/SearchAndSort/SearchAndSortQuery.js

```diff
@@ -397,7 +397,7 @@ class SearchAndSortQuery extends React.Component {
   };
 
   onResetSearch = (submit = true) => {
-    const submitCallback = submit ? () => { this.onSubmitSearch(); } : noop;
+    const submitCallback = submit ? () => { this.onSubmitSearch(false, 'search.before-reset'); } : noop;
```

Comment:
Would `null` be more expressive than `false` here? Hmmm, looks like we already use `false` in other places (line 252) but this seems unfortunate since the check in `onSubmitSearch` is really evaluating an object reference rather than checking a boolean.